### ISSUE TITLE
LearningRateScheduler doesn't work

### ISSUE THREAD

**Issue by ruriohead** [divamgupta/image-segmentation-keras#138]
I modify train.py and added callbacks property.
Then, I tried to use LearningRateScheduler just like beneath.

```
def step_decay(epoch,lr):
      print(epoch,lr)
      return lr * 0.9 **epoch

lr_decay = LearningRateScheduler(step_decay)

```
However, my model just keep showing its learning rate as its initial one ( 0.01) and the epoch value is always 1.
I doubt if there is a problem with passing values to next epoch.

This is my model.
```
cb_my = LossHistory()
earlystop = EarlyStopping(monitor="val_dice_coef",patience = 1,verbose=1)

model = keras_segmentation.models.unet.vgg_unet(n_classes=12,input_height = 384,input_width= 384)
model.train(
    train_images = "./sip/seg_data/limit_test/30_train_target",
    train_annotations = "./sip/seg_data/limit_test/30_train_seg",
    checkpoints_path = "./sip/set_data",
    epochs = 50,
    batch_size =8,
    validate=True,
    val_images = "./sip/seg_data/limit_test/mini_test_target",
    val_annotations = "./sip/seg_data/limit_test/mini_test_seg",
    val_batch_size =10,
    steps_per_epoch = 256,
    callbacks = [cb_my,earlystop],
    optimizer_name = "rmsprop",
    metrics = ["accuracy",dice_coef],
    loss_function_name = categorical_focal_loss()
)
```
